esp/clients/ws_dfsclient/ws_dfsclient.cpp

```diff
@@ -861,6 +861,7 @@ class CLocalOrDistributedFile: implements ILocalOrDistributedFile, public CInter
             lfn.allowOsPath(true);
         if (!lfn.setValidate(fname))
             return false;
+        bool write = isWrite(accessMode);
```

Comment:
Any way to be sure that there are no other similar undetected issues introduced by this PR?

Reply:
I checked through again for every function that had a wait parameter.  Something to be wary of in the future.